ui/dashboardApp/index.ts

```diff
@@ -75,12 +84,60 @@ async function main() {
     if (!routing.isLocationMatchPrefix(auth.signInRoute)) {
       if (!auth.getAuthTokenAsBearer()) {
         singleSpa.navigateToUrl('#' + auth.signInRoute)
-        return
       }
     }
   })
 
   singleSpa.start()
 }
 
-main()
+/////////////////////////////////////
+
+// check whether this is a portal page
+function checkPortal(): boolean {
+  return window.location.pathname.endsWith('/portal')
+}
```

Comment:
I prefer to put them in `routing` and use something like `routing.isLocationMatch`.

Reply:
Yep! will update it soon.